src/dali/plugin/pair_converter/ccxt_binance_locked.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2023 Neal Chambers
```

Comment:
Just a nit: why not call this just ccxt_binance.py: is there ever going to be a ccxt_binance.py that is not locked?